armotypes/portaltypesutils.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 // Getters & Setter used by derived types for interfaces implementation
-func (p *PortalBase) GetGuid() string {
+func (p *PortalBase) GetGUID() string {
```

Comment:
Revert back breaking change from https://github.com/armosec/armoapi-go/pull/120/files#diff-640e9bd118000dc58abf8ad5de45069b711f212c3586a516752fc822191caaebR9
Talked with Yischa and we will fix the dashboard interface

Reply:
are we ok to merge before the fix in dashboard?